src/qibolab/dummy.py

```diff
@@ -36,6 +36,7 @@ def create_dummy(with_couplers: bool = True):
     twpa_pump.power = 10
 
     runcard = load_runcard(pathlib.Path(__file__).parent / "dummy.yml")
+    extras_folder = pathlib.Path(__file__).parent / "dummy/"
```

Comment:
```suggestion
    extras_folder = pathlib.Path(__file__).parent / "dummy"
```
Following @AleCandido suggestion down below